.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -51,33 +49,27 @@ body:
         2.
         3.
         ...
-      render: bash
     validations:
       required: true
 
   - type: textarea
-    id: Expected
+    id: expected_behaviour
     attributes:
       label: Expected behaviour
       description: A concise description of what you expected to happen.
-      render: bash
     validations:
       required: true
 
   - type: textarea
-    id: Actual
+    id: actual_behaviour
     attributes:
       label: Actual behaviour
       description: A concise description of what you're experiencing.
-      render: bash
```

Comment:
I think removing `render: bash` should allow users to use Markdown formatting so they can embed links and attach screenshots